### ISSUE TITLE
No way to control the Java class attributes of the Constants interface

### ISSUE THREAD

**Issue by janssen** [swig/swig#409]
There's a way to control the module class attributes, and the JNI class attributes, but no way to make the constants class "protected" instead of "public".

**Comment by ojwb** [comment]
`java.cxx` still seems to hardcode this in git master:

```
      Printf(f_module, "public interface %s {\n", constants_interface_name);
```

**Comment by ojwb** [comment]
This was implemented last month by 51b281eff092b074b3780d056444a9dc4f8b82b2.
